webpack.config.js

```diff
@@ -47,13 +44,40 @@ const config = [
       // bundles aren't duplicating libs.
       new webpack.optimize.CommonsChunkPlugin({
         name: 'vendor',
-        minChunks: module => module.context && module.context.indexOf('/node_modules/') > -1,
+        minChunks(module) {
+          if ((/TinyMCE_/).test(module.resource)) {
+            return false;
+          }
+          return module.context && module.context.indexOf('/node_modules/') > -1;
+        },
       }),
     ],
     watchOptions: {
       poll: true
     }
   },
+  {
+    name: 'tinymce',
+    entry: {
+      TinyMCE_sslink: `${PATHS.LEGACY_SRC}/TinyMCE_sslink.js`,
+      'TinyMCE_sslink-external': `${PATHS.LEGACY_SRC}/TinyMCE_sslink-external.js`,
+      'TinyMCE_sslink-email': `${PATHS.LEGACY_SRC}/TinyMCE_sslink-email.js`,
+    },
+    output: {
```

Comment:
Let's reduce the duplication here. Everything from this point down can be in a common object. Then, for both configs, use `...commonConfig`. For the vendor bundle, use

```js
...commonConfig,
plugins: [
  ...commonConfig.plugins,
  new webpack.optimise.CommonChunks(....
]
```